apps/hexagon_matmul/adb_run_on_device.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
Do we need multiple copies/versions of adb_run_on_device.sh? Could we refactor into a single one?

Reply:
I think the best we could do is pull the Hexagon runtime push stuff to a separate script. However, it's messy because it's hard to share environment variables between the scripts (I do this in our internal code that has similar scripts and I've never been happy with it).